class/Integration/Gutenberg.php

```diff
@@ -79,4 +83,18 @@ public function is_gutenberg_active() {
 
 	}
 
+	/**
+	 * Checks if is Quick Edit action
+	 *
+	 * @since [Next]
+	 * @return boolean
+	 */
+	public function is_quick_edit() {
+		$edit_action = isset( $_POST['action'] ); //phpcs:ignore WordPress.Security.NonceVerification.Missing
```

Comment:
Is this really working? `isset` returns bool not the value

Reply:
yeah, it's working perfectly because it will be in most cases true :P I've fixed that :)